multinode/bootstrapping.md

```diff
@@ -0,0 +1,190 @@
+# Data Node Bootstrapping [](distdb-bootstrap)
+
+Scaling up TimescaleDB is achieved by adding data nodes to an access
+node using the [`add_data_node`][add_data_node] and
+[`attach_data_node`][attach_data_node] functions. In order for a data
+node to be available to the access node it is necessary to ensure that
+it is properly setup to work as a data node. This process is referred
+to as *bootstrapping the data node*, which is done automatically each
+time a data node is added to an access node (unless `bootstrap` is
+false, in which case it just verifies that the data node is properly
+set up).
+
+The steps performed when boostrapping a data node are:
+
+1. Connect to the data node and authenticating the user.
+1. Check that there is a database on the data node and create it
+   otherwise.
+2. Check that the Timescale extension is available in the database or
+   create it otherwise.
+3. Set metadata on the data node.
+
+## Authentication during bootstrapping
+
+All connections to the data node are made as the current user on the
+access node. Which requires that:
+
+1. The current user exists on both the data node and access node.
+2. The data node is set up to handle logins using either certificates
+   or password files, as described in the [Data Node Authentication
+   Chapter][distdb-auth].
+
+## Adding data nodes to the access node
+
+A new data node is added to the access node using the
+[`add_data_node`][add_data_node] command.
+
+```
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com');
+```
+
+The access node has to ensure that the database on the data node is
+properly configured by either bootstrapping the database or verifying
+that the database exists and is correctly configured. This is
+controlled with the `bootstrap` option, which can be either `true`
+(indicating that the data node should be bootstrapped) or `false`
+(indicating that the data node will only be validated, but no
+bootstrapping will take place).
+
+```
+SELECT * FROM add_data_node('data_node_1', 'data1.example.com',
+                            boostrap => false);
+```
+
+The default value for the `bootstrap` option is `true`, which means
+that the data node is bootstrapped when added.
+
+## Setting up the database on the data node
+
+The database to bootstrap on the data node is given by the name of the
+current database on the access node. For example, if you are connected
```

Comment:
I don't understand this sentence. Is the database given or is the name fo the current database? I think it is either given or it uses the current database by default. Suggestion:
```suggestion
The database to bootstrap on the data node will have the name of the
  current database on the access node by default. For example, if you are connected
```

Reply:
Fixed.